test/integration/query/selects.js

```diff
@@ -150,6 +150,78 @@ module.exports = function (knex) {
         });
     });
 
+    it('#4199 - adheres to optimizer hints', async function () {
+      const expectedErrors = {
+        mysql: {
```

Comment:
can you also implement this at least for PostgreSQL?

Reply:
@kibertoad  Vanilla PostgreSQL does not suport optimizer hints of this kind. It is supported just in extensions like [EDB Postgres Advanced Server](https://www.enterprisedb.com/edb-docs/d/edb-postgres-advanced-server/user-guides/database-compatibility-for-oracle-developers-guide/11/Database_Compatibility_for_Oracle_Developers_Guide.1.038.html
).